docs/guide-end-of-form.rst

```diff
@@ -0,0 +1,243 @@
+
+.. _guide-form-states-common-workflows:
+
+Customizations for common workflows
+------------------------------------
+
+Some questions to ask yourself as you design your workflow are:
+
+* Is it possible for a data collector to reach the end of the form but still have information to fill in?
+* What should happen if a data collector is interrupted while filling out a form?
+* How capable are data collectors of remembering a workflow?
+* How many times will data collectors repeat the same workflow?
+* How capable are data collectors of making independent decisions when faced with unexpected situations like an interview being interrupted?
+* How trusted are data collectors? Are they likely to want to "cheat" in some way to save time and/or effort?
+* What are the consequences of incorrect data being submitted? What are the next steps if that happens and is detected?
+
+As you answer these questions, you will get a clearer sense of what needs to happen when data collectors need to exit a form. This section includes some common workflow patterns and how to use the tools outlined above to support them.
+
+No edits allowed after leaving data collection subject
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In many workflows, it's important to guarantee that data is not changed after the data collector no longer has access to the data collection subject. For example, a nurse administering a vaccine should generally capture all data about that vaccination encounter while their patient is with them. They should not rely on their memory to fill in details after the encounter. To guarantee that data collectors have to fill out the form in one session:
+
+* Remove option to save draft from the form end screen and from within the form filling experience.
+* Hide the Drafts button from the landing screen.
+* (Generally) Turn on :guilabel:`Auto send`
+* (If data is highly sensitive or devices are not trusted) Turn on :guilabel:`Delete after send`
+* (If data collection must be linear) :ref:`Disable moving backwards <moving-backwards-setting>`
+* (If data collectors may be tempted to change settings) Set an admin password and hide :guilabel:`User Settings` set above
+
+When data collectors reach the form end screen, they will only have the option to finalize. If they are interrupted during a form filling session, they will need to exit and discard changes or rely on automatic data backups and recovery (the partially-filled form will open automatically when they open the same blank form again).
+
+Edits are encouraged until a certain point in time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In some workflows, new information may need to be added to a form after a first data collection event. For example:
+
+* a form may capture multiple days' worth of interviews
+* a data collection subject such as a mountain may be observed from different angles at different times, revealing new information
```

Comment:
Mountain is so oddly specific and weird. Drop it.

data might be collected on a subject at different times, revealing new information